generators/client/templates/react/src/main/webapp/app/modules/administration/gateway/gateway.tsx.ejs

```diff
@@ -90,16 +90,16 @@ export class GatewayPage extends React.Component<IGatewayPageProps> {
           <p>
             <Button onClick={this.gatewayRoutes} color={isFetching ? 'danger' : 'primary'} disabled={isFetching}>
               <FontAwesomeIcon icon="sync" />&nbsp;
-              <Translate component="span" contentKey="health.refresh.button" />
+              <Translate component="span" contentKey="health.refresh.button">Refresh</Translate>
```

Comment:
These are supposed to be replaced with actual content during the translation stripping step, isnt that working?